pkg/scaffold/v2/makefile.go

```diff
@@ -98,7 +98,7 @@ generate: controller-gen
 docker-build: test
 	docker build . -t ${IMG}
 	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${IMG}"'@' ./config/default/manager_image_patch.yaml
+        cd config/manager && kustomize edit set image controller=${IMG}
```

Comment:
nit: please fix the indentation. We should use "\t" here.

Reply:
I am able to run it fine on my local 

```
cd testdata/project-v2
kustomize build config/default
```